index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome Books</title>
+
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="container header">
+      <p class="page-title">Awesome Books</p>
+      <nav>
+        <ul class="nav-links">
+          <li class="nav-link nav-link--active" data-sect="1">List</li>
+          <li class="dash">|</li>
+          <li class="nav-link" data-sect="2">Add new</li>
+          <li class="dash">|</li>
+          <li class="nav-link" data-sect="3">Contact</li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] As shown in the wireframe, please use luxon library to show the current date-time below the navigation bar.